config.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+options:
+  config_file:
+    type: string
+    default: ""
```

Comment:
Can we provide a sensible default here?

Reply:
The blackbox exporter has a default configuration, so you don't need to pass a configuration file :) It's the same way of something like alertmanager, where the workload will function even with an empty config file variable!